deployments/upgrades/export_configs_to_helm.py

```
@@ -0,0 +1,339 @@
+        parser.error('OSMO_URL or --url is required')
+
+    base_url = args.url.rstrip('/')
+    headers = {}
+    if args.token:
+        headers['Authorization'] = f'Bearer {args.token}'
+    for header in args.header:
+        key, _, value = header.partition(':')
+        headers[key.strip()] = value.strip()
+
+    print(f'# Exported from {base_url}', file=sys.stderr)
+    print(f'# Paste this into services.configs in your Helm values.\n',
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)